auslib/web/admin/views/rules.py

```diff
@@ -117,10 +118,13 @@ def _post(self, id_or_alias, transaction, changed_by):
 
         what, mapping_values, fallback_mapping_values = process_rule_form(connexion.request.get_json())
 
-        if what.get('mapping', None) is not None and len(mapping_values) != 1:
+        if 'mapping' in what and not what.get('mapping', None):
+            return problem(400, 'Bad Request', 'mapping value cannot be set to null/empty')
+
+        elif what.get('mapping', None) and len(mapping_values) != 1:
             return problem(400, 'Bad Request', 'Invalid mapping value. No release name found in DB')
 
-        if what.get('fallbackMapping', None) is not None and len(fallback_mapping_values) != 1:
+        elif what.get('fallbackMapping') is not None and len(fallback_mapping_values) != 1:
```

Comment:
No we won't due to this [line](https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/rules.py#L30) , I'll have to edit it to move it [here](https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/rules.py#L45) so that blank field values are invalid for mapping but not for fallbackMapping. Should I do it ?